src/stream/src/task/stream_manager.rs

```diff
@@ -347,13 +347,10 @@ impl LocalStreamManager {
     }
 
     /// Force stop all actors on this worker, and then drop their resources.
-    pub async fn stop_all_actors(&self) -> StreamResult<()> {
+    pub async fn stop_all_actors(&self) {
         self.core.lock().await.stop_all_actors().await;
-        self.reset_barrier_manager();
```

Comment:
Did this rest operation delete by mistake?

Reply:
Thank you so much.